DeepSpeech.ipynb

```diff
@@ -615,13 +618,19 @@
    "source": [
     "def calculate_accuracy_and_loss(batch_set):\n",
     "    # Obtain the next batch of data\n",
-    "    batch_x, batch_y, n_steps = batch_set.next_batch()\n",
+    "    batch_x, batch_y = batch_set.next_batch()\n",
+    "\n",
+    "    # Obtain an input tensor for the batch, to be used when exporting the trained model\n",
+    "    batch_x_tensor = variable_on_cpu('batch_x', batch_x.shape)\n",
```

Comment:
We've been discussing this on IRC, but just a summary/recap for the issue;

- Inputs don't have to be placeholders
- This code assigns the content of batch_x to batch_x_tensor immediately after creation and names that resulting tensor for input
- You're right about the shape, need to fix that